app/models/consultation.rb

```diff
@@ -328,12 +339,16 @@ def application_link
 
   def period_days
     if environment_impact_assessment.try(:required?)
-      Consultation::EIA_PERIOD
+      Consultation::EIA_PERIOD_DAYS
     else
-      Consultation::DEFAULT_PERIOD
+      Consultation::DEFAULT_PERIOD_DAYS
     end
   end
 
+  def default_consultee_response_period
+    (environment_impact_assessment.try(:required?) && EIA_PERIOD) || DEFAULT_PERIOD
+  end
+
```

Comment:
I can't see if this is still being used anywhere?

Reply:
Yes, forgot to delete thanks